doc/internal/hackers-guide.asciidoc

```diff
@@ -122,6 +131,9 @@ crude menus that are somehow hard to use.
 
 === Keep a good balance of customizability
 
+**Status of this section:** the idea holds, but Newsboat arguably failed this
+goal (January 2021).
+
 The problem with user wishes is that too many people demand a possibility to
 customize this bell or that whistle within newsbeuter. Often, these
```

Comment:
Not sure whether this is on purpose, to signal the age, but it's still called "newsbeuter". The same goes for a few other lines I'm not able to comment on. :-)

Reply:
It's on purpose: I didn't want to get distracted updating the sections that aren't relevant to this particular PR. I'm also not yet sure how I want the document to look, so I don't have an immediate plan to rephrase all the old sections and update them.

Perhaps I should explicitly say "outdated", to make it even more clear that the sections wasn't updated?

Reply:
I'm absolutely fine with leaving it like that, no need to change anything. Just wanted to double-check.